sdk/eventhubs/azure_messaging_eventhubs/src/consumer/mod.rs

```diff
@@ -389,21 +386,18 @@ impl ConsumerClient {
     /// }
     /// ```
     #[tracing::instrument]
```

Comment:
Let's either remove these for now (we need to finalize the tracing story, and I've gotten some pushback on `tracing`) or change all instances to `#[tracing::instrument(skip_all, err)]`. It could leak secrets otherwise. Also, the client shouldn't implement `Debug` or `Display` - the former of which you'd have to have done to satisfy this tracing attribute as-is.